packages/nodes-base/nodes/Medium/Medium.node.ts

```diff
@@ -0,0 +1,448 @@
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'post',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Create a post',
+						value: 'create',
+						description: 'Create a post on the user authenticated profile',
+					},
+
+				],
+				default: 'create',
+				description: 'The operation to perform.',
+			},
+
+			// ----------------------------------
+			//         post:create
+			// ----------------------------------
+			{
+				displayName: 'Publication',
+				name: 'publication',
+				type: 'boolean',
+				default: false,
+				description: 'Are you publishing under a publication?'
+			},
+			{
+				displayName: 'Publication ID',
+				name: 'publicationId',
+				type: 'options',
+				displayOptions: {
+					show: {
+						publication: [
+							true,
+						],
+					},
+				},
+				typeOptions: {
+					loadOptionsMethod: 'getPublications',
+				},
+				default: '',
+				description: 'Publication ids',
+			},
+			{
+				displayName: 'Title',
+				name: 'title',
+				type: 'string',
+				default: '',
+				placeholder: 'My open source contribution',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: [
+							'create',
+						],
+						resource: [
+							'post',
+						],
+					},
+				},
+				description: 'Title of the post. It should be less than 100 characters',
+			},
+			{
+				displayName: 'Content format',
+				name: 'contentFormat',
+				default: '',
+				placeholder: 'My open source contribution',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: [
+							'create',
+						],
+						resource: [
+							'post',
+						],
+					},
+				},
+				type: 'options',
+				options: [
+					{
+						name: 'HTML',
+						value: 'html',
+					},
+					{
+						name: 'Markdown',
+						value: 'markdown',
+					},
+
+				],
+
+				description: 'The format of the content to be posted.',
+			},
+			{
+				displayName: 'Content',
+				name: 'content',
+				type: 'string',
+				default: '',
+				placeholder: 'My open source contribution',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: [
+							'create',
+						],
+						resource: [
+							'post',
+						],
+					},
+				},
+				description: 'The body of the post, in a valid, semantic, HTML fragment, or Markdown.',
+			},
+			{
+				displayName: 'Additional Fields',
+				name: 'additionalFields',
+				type: 'collection',
+				placeholder: 'Add Fields',
+				displayOptions: {
+					show: {
+						operation: [
+							'create',
+						],
+						resource: [
+							'post',
+						],
+					},
+				},
+				default: {},
+				options: [
+
+					{
+						displayName: 'Tags',
+						name: 'tags',
+						type: 'string',
+						default: '',
+						placeholder: 'open-source,mlh,fellowship',
+						description: 'Tags separated by comma to classify the post. Only the first three will be used. Tags longer than 25 characters will be ignored.',
```

Comment:
Suggestion: `Comma-separated strings to be used as tags for post classifications. Max allowed tags: 3. Max tag length: 25 characters.`